cases/PFEM_Metafor/birdImpact_deformable_panel_bird_Pfem_Smojver_Axisym.py

```diff
@@ -33,15 +16,12 @@
 w = None
 
 class Module:
```

Comment:
FYI: that is the class I'm going to remove with smart pointers

Reply:
Good to know, because I am doing the same thing!
I will change that tomorrow.